composer.json

```diff
@@ -69,13 +70,29 @@
         }
     },
     "scripts": {
-        "post-update-cmd": [
-            "\\PhpPact\\Standalone\\Installer\\InstallManager::uninstall"
-        ],
         "start-provider": "php -S localhost:58000 -t example/src/Provider/public/",
         "static-code-analysis": "phpstan analyse src/ --level=5",
         "lint": "php-cs-fixer fix --config .php-cs-fixer.php --dry-run",
         "fix": "php-cs-fixer fix --config .php-cs-fixer.php",
         "test": "phpunit --debug -c example/phpunit.all.xml"
+    },
+    "extra": {
+        "downloads": {
+            "pact-ruby-standalone": {
+                "version": "1.91.0",
+                "variables": {
+                    "{$os}": "PHP_OS_FAMILY === 'Windows' ? 'win32' : (PHP_OS === 'Darwin' ? 'osx' : 'linux')",
+                    "{$architecture}": "PHP_OS === 'Linux' ? '-x86_64' : ''",
+                    "{$extension}": "PHP_OS_FAMILY === 'Windows' ? 'zip' : 'tar.gz'"
+                },
+                "url": "https://github.com/pact-foundation/pact-ruby-standalone/releases/download/v{$version}/pact-{$version}-{$os}{$architecture}.{$extension}",
+                "path": "bin/pact-ruby-standalone"
+            }
+        }
+    },
+    "config": {
+        "allow-plugins": {
+            "tienvx/composer-downloads-plugin": true
+        }
```

Comment:
Add this config to make running test easier. I think each project depend on this library also need to define this config again in `composer.json`

Reply:
I think if we can provide support for a user to support the ffi being provided by them in their own location (which should be fine as we can have to provide a path to the library loader) we can support users who dont want to, or cant enable scripts to download.

Reply:
I agree with it.

This feature can be useful for plugins. See this [link](https://pact-foundation.slack.com/archives/C047TCR7B6W/p1679881619953729) for explanation.

Currently I don't have any idea on how to do it. Can you create a ticket so we can look at it later?

Reply:
of course buddy 👍🏾  shall do now